### PR TITLE
Delete at most one resource policy at a time

### PR DESCRIPTION
**Pull request by mr0re1**
* Delete at most one resource policy at a time
* Don't fail if some policy can't be deleted

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh
Comment (nick-stroud) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh:1]: In addition to running integration tests please manually test for a case with 2 active placement policies.

Comment (nick-stroud) [inline community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh:1]: If this is hotfix to main and there is code change then we also need to roll the patch version number.

**Review by nick-stroud — APPROVED** [review]

File: community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh
Comment (mr0re1) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh:1]: done

Comment (mr0re1) [reply community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh:1]: ack

**Pull request merged**